docker-compose.yml

```diff
@@ -679,6 +680,45 @@ services:
       UBUNTU:
     command: *cpp-command
 
+  ubuntu-cpp-emscripten-python:
+    # Usage:
+    #   docker-compose build ubuntu-cpp-emscripten-python
+    #   docker-compose run --rm ubuntu-cpp-emscripten-python
+    # Parameters:
+    #   ARCH: amd64, arm64v8, ...
+    #   UBUNTU: 22.04
+    image: ${REPO}:${ARCH}-ubuntu-22.04-cpp-emscripten
+    build:
+      context: .
+      dockerfile: ci/docker/ubuntu-22.04-cpp-emscripten.dockerfile
```

Comment:
If we want to use Ubuntu based image for this, we should reuse `ci/docker/linux-apt-python-3.dockerfile` or create `ci/docker/linux-apt-python-3-emscripten.dockerfile` based on `linux-apt-python-3.dockerfile`.
If we need to use conda, we should create `ci/docker/conda-python-emscripten.dockerfile` based on `ci/docker/conda-python.dockerfile`.

Reply:
okay, done the conda way

Reply:
Could you update the path, `image:` and `cache_from:`?